.github/workflows/docker-tests.yml

```diff
@@ -11,13 +11,18 @@ concurrency:
 
 jobs:
   tests:
-    name: Run Install Script Validation
+    name: Validate Platform Installation
     runs-on: ubuntu-20.04
 
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Validate Installation
+      - name: Install Platform
         run: |
-          interactive=false clone_platform=false open_dashboard=false ./install-and-launch.sh
+          interactive=false clone_platform=false ./install-and-launch.sh
+
+      - name: Check Health
+        run: |
+          apt-get update && apt-get install -y yq
```

Comment:
Do we need `yq` or just `jq`? If `jq` the default images _might_ have it.

Reply:
Oh I didn't realize I made that copy mistake! Well the good news is that we don't need this so I just removed it outright!